c/consul/Dockerfile/1.8.3-ubi8/README.md

```diff
@@ -0,0 +1,43 @@
+# Building Consul
+```
+docker build -t consul .
+docker run -it consul:latest
+```
+
+# Deploying Consul on Openshift
+```
+$ oc new-project consul
+--> Now using project "consul" on server "https://api.sneha-ppc.cp.fyre.ibm.com:6443".
```

Comment:
Can you replace the server address with **https://api-<cluster-address>:6443** ?
As no one can access the address as you are not providing username & password.

Reply:
Updated.